### DISCUSSION TITLE
<how to do some thing on github>

### DISCUSSION THREAD

**Discussion by gavyeah8616** [community/community#119725] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

can i make a github page without a  organization and have it public just wanna know

**Comment by ebndev** [comment]
Hi @gavyeah8616, we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by aqiefsyahmi** [comment]
Yes, you can create a GitHub page for a personal project without needing to belong to an organization.

**Reply by gavyeah8616** [reply]
but can it be public or does it have to be private?

**Comment by eddysantiagoo** [comment, marked as answer]
@gavyeah8616 Yes, you can create a github pages from a private or public repository, and it is not necessary to have an organization to do that :)

If the answer helps you please mark this answer as the correct answer

**Comment by DhirajBembade** [comment]
Yes we can absolutely create a public GitHub Pages site without using an organisation.
